service/dap/server.go

```diff
@@ -286,34 +292,71 @@ func (s *Server) onInitializeRequest(request *dap.InitializeRequest) {
 	s.send(response)
 }
 
+// Output path for the compiled binary in debug or test modes.
+const debugBinary string = "./__debug_bin"
+
 func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
 	// TODO(polina): Respond with an error if debug session is in progress?
-	program, ok := request.Arguments["program"]
+
+	program, ok := request.Arguments["program"].(string)
```

Comment:
Sorry, I may be missing something here, but has the semantics of what's stored in `request.Arguments` changed?

Because before this change the `ok` was saying whether the key "program" is found in the map. After the change it's saying that the conversion to string is successful. Is this an intentional change?